lua/telescope/_extensions/project.lua

```diff
@@ -0,0 +1,54 @@
+local has_telescope, telescope = pcall(require, 'telescope')
+
+if not has_telescope then
+	error('This plugins requires nvim-telescope/telescope.nvim')
+end
+
+local actions = require("telescope.actions")
+local finders = require("telescope.finders")
+local pickers = require("telescope.pickers")
+local sorters = require("telescope.sorters")
+local builtin = require("telescope.builtin")
+
+local project = function(opts)
+	opts = opts or {}
+
+	local project_dirs = {["nvim config"] = "~/.config/nvim"}
+	local projects = {}
+
+	local search_selected_project = function(selection)
+		print(project_dirs[selection])
+		local selected_directory = project_dirs[selection]
+		builtin.find_files({cwd = selected_directory})
+	end
+
+	local n = 0
+	for k in pairs(project_dirs) do
+		n = n + 1
+		projects[n] = k
+	end
```

Comment:
You don't need n here. You can do `table.insert(projects, k)`

Reply:
Okay thanks, that's nicer. I haven't used Lua much, so learning a bit here.

Reply:
No problem. Most of us have no lua experience before writing neovim plugins.